Delicate-Butterflies/Bangazon-Site
# Add and Delete payment type jl

Related Issue: #7, #8 

Description:
User can now add and delete payment types

PR Testing Steps:
1. create a new user or use and existing one
2. click profile link in the nav
3. click the green "add payment type" button and fill out the form
4. you will redirect to the payment types list and can hit the delete button on any payment type
5. check the database to make sure the users payment types have the appropriate null or dates for the deletedAt column.

Check list before you make a PR:
